### ISSUE TITLE
unable to type Chinese after using backspace

### ISSUE THREAD

**Issue by zhaoyiping0622** [microsoft/terminal#5238]
# Environment

```none
Windows build number: 
Platform ServicePack Version      VersionString
-------- ----------- -------      -------------
 Win32NT             10.0.18363.0 Microsoft Windows NT 10.0.18363.0
Windows Terminal version (if applicable):
0.10.781.0
Any other software?
the built-in Chinese input method
```

# Steps to reproduce

open a windows terminal, type some characters when using Chinese input method(such as 'asdf'), but don't type space to confirm the input. type backspace to delete the characters. type some other characters and type space.

# Expected behavior

it can type Chinese characters.

# Actual behavior

it cann't type Chinese characters.

![wterror](https://user-images.githubusercontent.com/39238310/78453276-2fbf7c00-76c3-11ea-8812-23b9632ae3d2.gif)

**Comment by skyline75489** [comment]
I feel this is the same as #5054, which is fixed by #5109 but has not been released yet.

**Comment by zhaoyiping0622** [comment]
Duplicate of #5054 
thanks @skyline75489
